### ISSUE TITLE
Invalid HTML in Pivot - UL containing BUTTON directly w/o LI

### ISSUE THREAD

**Issue by BrendanMcK** [OfficeDev/office-ui-fabric-react#6504]
### Please provide a reproduction of the bug in a codepen:

Can see repro at [pivot page in Fabric docs](https://developer.microsoft.com/en-us/fabric#/components/pivot)

### Bug Report 
 
 
- __Package version(s)__: 6.49.2
- __Browser and OS versions__: any
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) no
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)
 
#### Describe the issue:

Inspect the rendered DOM tree for any Pivot instance - eg on the [Fabric docs page](https://developer.microsoft.com/en-us/fabric#/components/pivot)

![image](https://user-images.githubusercontent.com/1754373/46189554-ffb04500-c2a3-11e8-94ac-977c1b0899fd.png)

Note that we have `UL` that directly contains several `BUTTON` elements. This is invalid HTML - `UL` are only permitted to contain `LI` elements. This causes pages containing pivots to fail validation, which in turn fails [WCAG 2.0 4.1.1](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#ensure-compat-parses).

Couple of options for fixing this: if we want to keep the `UL`, then may need to add child `LI` elements wrapping each `BUTTON` - perhaps with `role=presentation`? - or replace the `UL` with a plain `DIV`.
